simple_api/django_object/validators.py

```diff
@@ -0,0 +1,29 @@
+from django.core.exceptions import ValidationError
+from simple_api.object.validators import Validator
+
+
+class DjangoValidator(Validator):
+    def __init__(self, field_name, validation_fn):
+        self.fn = validation_fn
```

Comment:
`self.callable = callable` - nemusi to byt nutne funkce

Reply:
Přejmenováno